batch/batch/worker/worker.py

```diff
@@ -361,6 +365,18 @@ async def run(self, worker):
                 self.container = await docker_call_retry(MAX_DOCKER_OTHER_OPERATION_SECS, f'{self}')(
                     create_container, config, name=f'batch-{self.job.batch_id}-job-{self.job.job_id}-{self.name}')
 
+            c = await docker_call_retry(MAX_DOCKER_OTHER_OPERATION_SECS, f'{self}')(self.container.show)
+
+            merged_overlay_path = c['GraphDriver']['Data']['MergedDir']
+            assert merged_overlay_path.endswith('/merged')
+            self.overlay_path = merged_overlay_path[:-7].replace(LOCAL_SSD_MOUNT, '/host')
+            os.makedirs(f'{self.overlay_path}/', exist_ok=True)
```

Comment:
can you remind me the purpose of this block?

Reply:
I want to add all of the container files it writes to the quota tracking. So I get the location of the merged dir and then figure out the root overlay path from that.